infrastructure/router/router.go

```diff
@@ -0,0 +1,22 @@
+package routes
+
+import (
+	"github.com/Le-MaliX/ACADEMY-GO-Q42021/interface/controllers"
+	"github.com/gin-gonic/gin"
+)
+
+func Routes() *gin.Engine {
+	r := gin.Default()
+
+	r.GET("/", func(c *gin.Context) {
```

Comment:
Even if this function is very small, it would be cleaner to have it defined within the controller package.